Add metadata to action to allow extending the library (#37)
woylie/let_me: README.md

]
```

> Metadata can be defined on an `action` for use in extending LetMe's
> functionality that are out of scope for the library itself.

Reviewer: ```suggestion
You can also define metadata on an `action`, which can be useful to extend
the functionality of the library.
```

> For example, you may want to expose some of your actions through your Absinthe
> GraphQL schema. Perhaps you wanted to exclude only certain actions. Using the
> `action`'s metadata, it becomes easier to do so.
> 
> With `metadata`, there should be no limit to how the library can be extended
> in your own application.

Reviewer: ```suggestion
For example, you may want to expose some of your actions through your Absinthe
GraphQL schema, but exclude certain actions from the schema. You could solve
this by adding a `:gql_exclude` key to the metadata.
```
